net2/BroDetect.js

```diff
@@ -1027,33 +1027,37 @@ module.exports = class {
         Number(obj.orig_bytes),
         Number(obj.resp_bytes)
       ];
+
+      const tmpspec = {
```

Comment:
`tmpspec` and `this.flowstash[flowspecKey]` (`flowspec`) is basically the same. Merge these two to reduce confusion